co-authors-plus.php

```diff
@@ -1289,14 +1291,16 @@ function filter_views( $views ) {
 				'author_name'           => wp_get_current_user()->user_nicename,
 			);
 		if ( 'post' != get_post_type() ) {
-			$mine_args['post_type'] = get_current_screen()->post_type;
+			$mine_args['post_type'] = get_post_type();
 		}
 		if ( ! empty( $_REQUEST['author_name'] ) && wp_get_current_user()->user_nicename == $_REQUEST['author_name'] ) {
 			$class = ' class="current"';
 		} else {
 			$class = '';
 		}
-		$views['mine'] = $view_mine = '<a' . $class . ' href="' . esc_url( add_query_arg( array_map( 'rawurlencode', $mine_args ), admin_url( 'edit.php' ) ) ) . '">' . __( 'Mine', 'co-authors-plus' ) . '</a>';
+
+		$count = sprintf(' <span class="count">(%s)</span>', number_format_i18n( $numposts ) );
+		$views['mine'] = $view_mine = '<a' . $class . ' href="' . esc_url( add_query_arg( array_map( 'rawurlencode', $mine_args ), admin_url( 'edit.php' ) ) ) . '">' . __( 'Mine' ) . $count . '</a>';
```

Comment:
Is there a reason why the `co-authors-plus` string was removed from the domain parameter of `__( 'Mine', 'co-authors-plus' )`?

Reply:
Yes - it seemed unnecessary to me. `Mine` is also a string from WP Core, and it made sense to me to use its translation. This would avoid the (possible) issue of someone translating CAP's `Mine` in a different way from Core, and end up with the string being displayed sometimes in one way, sometimes another :)

Reply:
Fair enough, carry on!